tst/Test/DBSetup.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Test.DBSetup where
+
+import           Gonimo.Server.Auth
+
+import           Gonimo.Server.DbEntities
+import           Gonimo.Server.Types
+import           Database.Persist.Sqlite  as Db
+
+import           Data.Time.Clock          (getCurrentTime)
+
+setupDB :: IO AuthData
+setupDB = do
+  now <- getCurrentTime
+  runSqlite "WAL=off :memory:" $ do
```

Comment:
If you experience problems, that something is not in the db you think should be there - try not using :memory:.